web/styles/popovers.css

```diff
@@ -833,14 +833,48 @@ ul {
         margin: 0;
     }
 
-    /* Override the default border-radius to properly align
-       the button corners with `stream_header_colorblock`. */
-    .dropdown-toggle {
-        border-radius: 1px 4px 4px 1px !important;
+    .move-topic-dropdown {
+        display: flex;
+        justify-content: flex-start;
+        margin-bottom: 10px;
+
+        .dropdown-widget-button {
+            /* Override the default border-radius to properly align
+            the button corners with `stream_header_colorblock`. */
+            border-radius: 1px 4px 4px 1px !important;
+            outline: none;
+            line-height: 24px;
+            width: auto;
+            max-width: 206px;
+        }
+
+        .stream_header_colorblock {
+            margin: 0;
+        }
+
+        #move_topic_stream_name {
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+            color: var(--color-text-default);
+
+            .stream-privacy-type-icon {
+                font-size: 13px;
+                width: 13px;
+                height: 13px;
+                position: relative;
```

Comment:
It is possible to use flexbox for positioning this icon instead of relative positioning?

Reply:
We can but it is still mispositioned by `0.5px`, so I guess this is better.

Reply:
0.5px how? That seems like something that ought to be able to be overcome, even in the absence of relative positioning :)

Reply:
<img width="149" alt="image" src="https://github.com/zulip/zulip/assets/25124304/7855424e-8a48-44bf-b705-30503c063f50">

I am using `display: flex` with `align-items: center`.